lib/custom_wizard/action.rb

```diff
@@ -20,10 +20,6 @@ def perform
       self.send(action['type'].to_sym)
     end
         
-    if creates_post? && @result.success?
-      @result.handler.enqueue_jobs
```

Comment:
@fzngagan Why are we removing this? What is replaced with?

Reply:
We're removing it because `enqueue_jobs` can be called on a `PostCreator` instance and not on a `NewPostManager` instance.

Also we only to explicitly call `enqueue_jobs` if we've created the post with `skip_jobs` param. Otherwise, the jobs are enqueued by default. 

https://github.com/discourse/discourse/blob/9c5ee4923b7b039162acdc5661da03d29688b428/lib/post_creator.rb#L234